.buildkite/pipeline.yml

```diff
@@ -0,0 +1,26 @@
+##################################
+# Docker image deployment pipeline
+##################################
+steps:
+  - label: Get docker tag and save it as metadata for use later
+    command: .buildkite/scripts/set_docker_tag_meta_data.sh
+
+  - wait
+
+  - label: Build docker image (master branches only)
+#    branches: master
+    command:
+      - .buildkite/scripts/build_tag_push_deployment_image.sh
+
+  - block: Human approval required to deploy docker image
+    branches: "*"
+    prompt: |
+      Clicking "OK" below will unblock this step and
+      permit the docker image to be tagged as latest
+      which means that the image will be used as a
+      base image for all runtime builds.
```

Comment:
Should this be "all contract kit builds" instead?

Reply:
We could either make building the docker image apply to all commits and only push as latest commits to master. Or we can build the image and push it as latest only on master. Eventually we could add tests here so that the pipeline not only builds the image but also tests it. I am thinking that we should build for all commits and push as latest only off of master. What do you think? In this case, the previous commented out line will be replaced with `branches: *`.

Reply:
Makes sense, might as well build for all commits.